main.tf

```diff
@@ -106,6 +107,7 @@ module "additional_tfe_workspaces" {
   global_remote_state            = each.value.global_remote_state
   name                           = coalesce(each.value.name, each.key)
   oauth_token_id                 = coalesce(each.value.vcs_oauth_token_id, var.tfe_workspace.vcs_oauth_token_id)
+  path                           = coalesce(each.value.path)
```

Comment:
```suggestion
  path                           = coalesce(each.value.path, var.tfe_workspace.path)
```

Reply:
This allows for only setting the path once and enable this for both workspaces if needed

Reply:
Thanks for pointing out. I missed that.